sdk/eventhub/Azure.Messaging.EventHubs/tests/Compatibility/TrackOneComparer.cs

```diff
@@ -147,7 +147,16 @@ internal static class TrackOneComparer
                 return false;
             }
 
-            return trackOneEvent.Properties.OrderBy(kvp => kvp.Key).SequenceEqual(trackTwoEvent.Properties.OrderBy(kvp => kvp.Key));
+            if (!trackOneEvent.Properties.OrderBy(kvp => kvp.Key).SequenceEqual(trackTwoEvent.Properties.OrderBy(kvp => kvp.Key)))
+            {
+                return false;
+            }
+
+            // Validate the runtime metrics properties.
+
+            return ((trackOneEvent.LastSequenceNumber != default ? trackOneEvent.LastSequenceNumber : default(long?)) == trackTwoEvent.LastPartitionSequenceNumber)
```

Comment:
Track One's `LastSequenceNumber` is a `long`, correct? I don't know exactly how Track One code works, but isn't it possible to be `zero` by chance? I still don't know how this and `offset` are supposed to behave.

Reply:
Not a big issue anyway, since we're using this method only in tests.

Reply:
So far as I'm aware, sequence number cannot be 0 for a legal value.   Offset can be, but I could swear that sequence number is 1-based.    Since this is track one and working in Live tests against brand new Event Hub instances, I'm inclined to say that I'm comfortable enough with it to leave it be until P4 when we remove the T1 dependency.